abci/types/types.proto

```diff
@@ -346,7 +347,7 @@ message Evidence {
   int64                     height    = 3;
   google.protobuf.Timestamp time = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
   int64                     total_voting_power = 5;
-  Validator                 complainant = 6; // Allowed to be null
+  int64                     threshold = 6; 
```

Comment:
No longer need to send the validator who signed the complaint as we make the evidence from each validator each aeon unique so that duplicates can not be submitted into the blocks